proto/controlclient.proto

```diff
@@ -15,6 +15,36 @@ service ScsControl {
   rpc CreateSigningKey (_CreateSigningKeyRequest) returns (_CreateSigningKeyResponse) {}
   rpc RevokeSigningKey (_RevokeSigningKeyRequest) returns (_RevokeSigningKeyResponse) {}
   rpc ListSigningKeys (_ListSigningKeysRequest) returns (_ListSigningKeysResponse) {}
+  rpc CreateIndex (_CreateIndexRequest) returns (_CreateIndexRequest) {}
+  rpc DeleteIndex(_DeleteIndexRequest) returns (_DeleteIndexResponse) {}
+  rpc ListIndexes(_ListIndexesRequest) returns (_ListIndexesResponse) {}
+}
+
+message _CreateIndexRequest {
+  string index_name = 1;
+  uint64 num_dimensions = 2;
+}
+
+message _CreateIndexResponse {
+}
+
+message _DeleteIndexRequest {
+  string index_name = 1;
+}
+
+message _DeleteIndexResponse {
+}
+
+message _ListIndexesRequest {
```

Comment:
@virratanasangpunth though, if you're gonna do pagination, you need an optional token param here, right?

Reply:
🦅 👀